src/navbar/navbar.styles.tsx

```diff
@@ -38,7 +38,7 @@ export const Nav = styled.nav<StyleProps>`
     transition: ${Transition.Base};
 
     ${MediaQuery.MaxWidth.tablet} {
-        height: 3.5rem;
+        height: ${() => NAVBAR_MOBILE_HEIGHT}rem;
```

Comment:
why does this need to be a function?

Reply:
hmm, this was originally meant to resolve a type error, but turns out the actual error is from using the constant before its declaration

will amend